fish-friends/src/components/Dashboard.js

```diff
@@ -30,19 +32,21 @@ class App extends Component {
             <div className="db-column db-cards">
 
                 <div className="db-navCardContainer">
-                    <Card className="db-navCard">
-                        <Link><CardText><AssignmentIcon /><button onClick={this.togglePopup.bind(this)}> Create Log</button></CardText></Link>
-                    </Card>
+                  <Card className="db-navCard">
+                    <Link onClick={this.togglePopup.bind(this)}>
+                      <CardText>
+                        <AssignmentIcon /> Create Log
```

Comment:
Did you mean to not have a button tag?

Reply:
"Refactored the popup form by removing the button and adding the bind to the Link."